uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.js

```diff
@@ -404,6 +404,7 @@ define([
         })
         .then(function (results) {
           vm.sections.expired.data = results.expiredBalanceChangesFlatten.concat(results.expiredTOILS);
+          vm.sections.expired.data = _.sortBy(vm.sections.expired.data, 'expiry_date');
```

Comment:
the expiry_date field is always a non empty string representation of a date using the `yyyy-mm-dd` format. So I think it's safe.